src/lib/molecules/nav.svelte

```diff
@@ -0,0 +1,141 @@
+<section class="p-menu1">
+    <nav id="navbar" class="navigation" role="navigation">
+      <input id="toggle1" type="checkbox" />
+      <label class="hamburger1" for="toggle1">
+        <div class="top"></div>
+        <div class="meat"></div>
+        <div class="bottom"></div>
+      </label>
+    
+      <nav class="menu1">
+        <a class="link1" href="/">Leeslijst</a>
+        <a class="link1" href="/uitleninge">Uitleningen</a>
+        <a class="link1" href="/">Uitleengeschiedenis</a>
+        <a class="link1" href="/betalingen">Betalingen</a>
+        <a class="link1" href="/">Reserveringen</a>
+        <a class="link1" href="/">Contact</a>
+      </nav>
+  </nav>
+  </section>
+  
+  <style>
+  
+  a {
+    color: var(--primary-light-color);;
+    text-decoration: none;
+  }
+  section {
+    position: absolute;
+    top: 0;
+    left: 9%;
+    width: 15vw;
+    position: fixed;
+  }
+    
+  /* Hamburger */
+  .hamburger1 {
+    display: grid;
+    justify-content: center;
+    grid-template-rows: repeat(3, 1fr);
+    padding: 2rem 3em;
+    z-index: 120;
+    background-color: var(--primary-accent-color);
+    border-radius: 0 0 1rem 1rem;
+  }
+  
+  .hamburger1 div {
+    background-color: var(--primary-light-color);;
+    position: relative;
+    width: 40px;
+    height: 5px;
+    margin-top: 7px;
+    -webkit-transition: all 0.2s ease-in-out;
+    transition: all 0.2s ease-in-out;
+  }
+  
+  #toggle1 {
+    display: none;
+  }
+  
+  #toggle1:checked + .hamburger1 .top {
+    -webkit-transform: rotate(-45deg);
+            transform: rotate(-45deg);
+    margin-top: 2.5px;
+    
+  }
+  
+  #toggle1:checked + .hamburger1 .meat {
+    -webkit-transform: rotate(45deg);
+            transform: rotate(45deg);
+    margin-top: -10px;
+    border-radius: 0;
+  }
+  
+  #toggle1:checked + .hamburger1 .bottom {
+    -webkit-transform: scale(0);
+            transform: scale(0);
+            border-radius: 0;
+  }
+  #toggle1:checked + .hamburger1 {
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 0;
+}
+  #toggle1:checked ~ .menu1 {
+    height: auto;
+  }
+  
+  
+  /* Menu */
+  .menu1 {
+    width: 100vw !important;
+    background-color: var(--primary-accent-color);
+    margin: 0;
+    display: -ms-grid;
+    display: grid;
+    grid-template-rows: 1fr repeat(4, 0.5fr);
+    grid-row-gap: 25px;
+    list-style: none;
+    clear: both;
+    width: auto;
+    padding: 0 1rem ;
+    height: 0px;
+    overflow: hidden;
+    transition: height .4s ease;
+    -webkit-transition: all 0.3s ease;
+    transition: all 0.3s ease;
+    z-index: 9;
```

Comment:
@EmonaSantiago kan je bij de overbodige code zoals webkit en '-ms-grid' checken of het weg kan?